ext/fluxnet_simulations/US-Var.jl

```diff
@@ -1 +1,157 @@
+"""
+    function get_domain_info(FT, ::Val{:US_Var}; dz_tuple = nothing,
+        nelements = 14, zmin = FT(-0.5), zmax = FT(0)
+Gets and returns primary domain information for the US-Var (California Vaira
+Ranch Ione) Fluxnet site.
 
+The data source comes from: Xu and Baldocchi, 2003.
+"""
+function get_domain_info(
+    FT,
+    ::Val{:US_Var};
+    dz_tuple = nothing,
+    nelements = 14,
+    zmin = FT(-0.5),
+    zmax = FT(0),
+)
+
+    return (
+        dz_tuple = dz_tuple,
+        nelements = nelements,
+        zmin = zmin,
+        zmax = zmax,
+    )
+end
+
+"""
+    get_parameters(::Val{:US_Var}; kwargs...)
+
+Gets parameters for the Fluxnet site US-Var (California Vaira Ranch Ione)
+and returns them as a Named Tuple.
+
+Data sources:
+
+Soil parameters:
+    - Bonan, G. Climate change and terrestrial ecosystem modeling. Cambridge University Press, 2019.
+    - Xu and Baldocchi 2003, doi:10.1016/j.agrformet.2003.10.004
+Conductance parameters:
+    - CLM 5.0 Tech Note: https://www2.cesm.ucar.edu/models/cesm2/land/CLM50_Tech_Note.pdf
+Photosynthesis parameters:
+    - CLM 5.0 Tech Note: https://www2.cesm.ucar.edu/models/cesm2/land/CLM50_Tech_Note.pdf
+    - Slevin et al. 2015, https://doi.org/10.5194/gmd-8-295-2015
+Hydraulics parameters:
+    - Holtzman, Nataniel, et al. 2023, https://doi.org/10.1029/2023WR035481
+    - Xu and Baldocchi 2003, doi:10.1016/j.agrformet.2003.10.004
+"""
+function get_parameters(
+    ::Val{:US_Var};
+    time_offset = 8,
+    lat = FT(38.4133),
+    long = FT(-120.9508),
+    atmos_h = FT(2),
+    soil_ν = FT(0.45),
+    soil_K_sat = FT(0.45 / 3600 / 100),
+    soil_S_s = FT(1e-3),
+    soil_vg_n = FT(2.0),
+    soil_vg_α = FT(2.0),
+    θ_r = FT(0.067),
+    ν_ss_quartz = FT(0.3),
+    ν_ss_om = FT(0.02),
+    ν_ss_gravel = FT(0.0),
+    z_0m_soil = FT(0.01),
+    z_0b_soil = FT(0.01),
+    soil_ϵ = FT(0.98),
+    soil_α_PAR = FT(0.2),
+    soil_α_NIR = FT(0.2),
+    Ω = FT(1.0),
+    ld = FT(0.5),
+    G_Function = ConstantGFunction(ld),
+    α_PAR_leaf = FT(0.11),
+    λ_γ_PAR = FT(5e-7),
+    τ_PAR_leaf = FT(0.05),
+    α_NIR_leaf = FT(0.35),
+    τ_NIR_leaf = FT(0.34),
+    ϵ_canopy = FT(0.97),
+    g1 = FT(166),
+    Drel = FT(1.6),
```

Comment:
A note that both Drel is universal and can be moved into ClimaParams at some point